lib/src/readers/chapter_reader.dart

```diff
@@ -31,8 +31,14 @@ class ChapterReader {
 
       EpubTextContentFileRef htmlContentFileRef;
       if (!bookRef.Content.Html.containsKey(contentFileName)) {
-        throw Exception(
-            "Incorrect EPUB manifest: item with href = \"${contentFileName}\" is missing.");
+        // It shouldn't be an encoded url, but we can handle it if it is.
+        final decoded = Uri.decodeFull(contentFileName);
```

Comment:
Personally, I am a fan of exiting early, it allows us to read the code top down, without having to go back to explore another code path. Can we do it like this?

```suggestion
        final decodedFileName = Uri.decodeFull(contentFileName);
        if (!bookRef.Content.Html.containsKey(decoded)) {
            throw Exception("Incorrect EPUB manifest: item with href = \"${contentFileName}\" is missing.");
        }
        
        contentFileName = decodedFileName
```
I have tested this code on my machine and it works but I did not copy and paste it here so please make sure to type it out and run the test if you accepts this suggestion.

This change has been made in https://github.com/orthros/dart-epub/pull/72/files, but I personally prefer this because it only decodes if it needs to.